orttraining/orttraining/test/python/orttraining_test_ortmodule_api.py

```diff
@@ -2014,6 +2014,7 @@ def test_bert_inputs_with_dynamic_shape():
     pt_model = _get_bert_for_sequence_classification_model(
```

Comment:
```suggestion
    pt_model = _get_bert_for_sequence_classification_model(is_training=True,
```
Having that on the API would be better for reuse. Maybe setting default to `False` as it is the official behavior of `BertForSequenceClassification.from_pretrained()`